api/v1alpha2/jenkins_webhook.go

```diff
@@ -17,6 +17,14 @@ limitations under the License.
 package v1alpha2
 
 import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/jenkinsci/kubernetes-operator/pkg/plugins"
+	"golang.org/x/mod/semver"
```

Comment:
```suggestion

	"golang.org/x/mod/semver"
```
We have a convention of separating our project imports.